### PR TITLE
Config and scripts required for Jenkins

### PR DESCRIPTION
**Pull request by sudeeshjohn**

### CODE REVIEW

File: config/prow/jenkins/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```
Comment (Rajalakshmi-Girish) [inline config/prow/jenkins/deploy.sh:1]: These steps in this script can be placed at the end of https://github.ibm.com/powercloud-secrets/secrets/blob/master/projects/k8s-prow/deploy.sh.
May be included as part of PR- https://github.ibm.com/powercloud-secrets/secrets/pull/25

File: config/prow/jenkins/update.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+kubectl apply -f jenkins-namespace.yaml 
+kubectl create -f jenkins-pvc.yaml --namespace jenkins
+
+export chart=jenkins/jenkins
+helm upgrade -f jenkins.yaml jenkins $chart  -n jenkins
+
```
Comment (Rajalakshmi-Girish) [inline config/prow/jenkins/update.sh:8]: In what case would we do a `helm upgrade`?
Also, hope namespace and pvc creation would be one time task before doing a `helm install`?

**Review by Rajalakshmi-Girish — CHANGES_REQUESTED** [review]

**Comment by mkumatag** [comment]
@sudeeshjohn @Rajalakshmi-Girish is this stale or can we address the comments and merge the PR?

**Comment by mkumatag** [comment]
@sudeeshjohn @Rajalakshmi-Girish any more updated?

File: config/prow/jenkins/jenkins-namespace.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: jenkins
```
Comment (Rajalakshmi-Girish) [inline config/prow/jenkins/jenkins-namespace.yaml:4]: @sudeeshjohn Can we please move this namespace resource to https://github.ibm.com/powercloud-secrets/secrets/blob/master/projects/k8s-prow/0001-namespaces.yaml?

Can be part of this PR https://github.ibm.com/powercloud-secrets/secrets/pull/29.

**Review by Rajalakshmi-Girish — CHANGES_REQUESTED** [review]

File: config/prow/jenkins/jenkins-namespace.yaml (hunk 3 above)
Comment (sudeeshjohn) [reply config/prow/jenkins/jenkins-namespace.yaml]: moved it to secret repo

**Review by Rajalakshmi-Girish — APPROVED** [review]

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**